elasticsearch/linux_distributions/opendistro-tar-install.sh

```diff
@@ -43,7 +43,7 @@ echo "done plugins"
 
 ##Check KNN lib existence in ES TAR distribution
 echo "Checking kNN library"
-FILE=$ES_KNN_LIB_DIR/libKNNIndexV1_7_3_6.so
+FILE=`ls $ES_KNN_LIB_DIR/libKNNIndex*.so`
```

Comment:
What will happen if there are multiple .so files? Can we detect that and fail the script?

Reply:
If you check 47, the test function would already fail it if there are more than 1.

Reply:
Well, it will print a warning. It won't fail the script.